pywheel/shim/_binwrapper.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+"""
+A generic wrapper to access nmodl binaries from a python installation
+Please create a softlink with the binary name to be called.
+"""
+import os
+from distutils import sysconfig
+import stat
+import sys
+from pkg_resources import working_set
+from distutils.ccompiler import new_compiler
+from distutils.sysconfig import customize_compiler, get_config_var
+
+
+def _set_default_compiler():
+    """Set (dont overwrite) CC/CXX so that apps dont use the build-time ones"""
+    ccompiler = new_compiler()
+    customize_compiler(ccompiler)
+    # xcrun wrapper must bring all args
+    if ccompiler.compiler[0] == 'xcrun':
+        ccompiler.compiler[0] = get_config_var("CC")
+        ccompiler.compiler_cxx[0] = get_config_var("CXX")
+    os.environ.setdefault("CC", ccompiler.compiler[0])
+    os.environ.setdefault("CXX", ccompiler.compiler_cxx[0])
```

Comment:
neueon nrnivmodl binary compiles c files and generate special and hence this was necessary. In case of nmodl, nmodl binary doesn’t compile generated c file (will be done by coreneuron). So this can be removed.